src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj

```diff
@@ -89,16 +95,12 @@
     </EmbeddedResource>
   </ItemGroup>
   <Target Name="GenerateFiles" AfterTargets="Build">
```

Comment:
I think this needs to follow the `ResolveSateliteResDeps` target defined in `CS_SDK.props`

Reply:
Ok, thanks!